mem_sql/pom.xml

```diff
@@ -259,12 +259,10 @@
                             <key>com.splicemachine.enableLegacyAsserts</key>
                             <value>true</value>
                         </systemProperty>
-                        <!-- Uncomment the following to dump activation class files for mem platform to the mem_sql directory.
```

Comment:
Please restore this, otherwise we are going to generate lots of files on mem builds

Reply:
Sorry, checked in by mistake, reverted.